api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>same a above, especially in the x-axis label, replace <code>Predicted</code> by <code>Fitted</code> to be clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>within-sample predictions are still predicted, just not out-of-sample predictions.</p><p>In statistics, it's common to plot the predicted versus actual data from a linear regression.</p>